src/dmd/globals.d

```diff
@@ -116,6 +116,7 @@ extern (C++) struct Param
     bool vtemplates;        // collect and list statistics on template instantiations
     bool vtemplatesListInstances; // collect and list statistics on template instantiations origins. TODO: make this an enum when we want to list other kinds of instances
     bool vgc;               // identify gc usage
+    bool vmemoize;          // memoize compilation
```

Comment:
not a v flag.
v means verbose and is for informational flags only.

Reply:
I see. Thanks.